converter.go

```diff
@@ -185,6 +186,9 @@ func snowflakeTypeToGo(ctx context.Context, dbtype snowflakeType, scale int64, f
 		}
 		return nil
 	case mapType:
+		if !structuredTypesEnabled {
```

Comment:
Is it sure thing that map won't be returned without enabling structured types? Doesn't fields need to be checked here?